install/windows.md

```diff
@@ -0,0 +1,76 @@
+
+If you like to run Airsonic as a Windows Service, download the [Non-Sucking Service Manager (NSSM)](https://nssm.cc/download).
+
+##### Install Airsonic
+
+Before we start, doublecheck if Java is working as expected and you setup the `PATH` environment variable right. Open a command prompt or the PowerShell console and type `java -version`. This should print the installed Java version.
+
+Create a new directory `C:\Program Files\Airsonic` and move the `airsonic.war` to this folder.
+
+Create an `airsonic.cmd` file in this folder and add the following content.
+
+```
+java.exe -Dairsonic.home="C:\\Program Files\\Airsonic" -Dserver.port=4040 -jar airsonic.war
+```
+
+Now you're ready! Start the `airsonic.cmd` file by double clicking on it, wait some seconds and browse to `http://localhost:4040`. You should see the Airsonic logon page.
+
+##### Installing Airsonic as Windows Service
+
+As it might be a bit inconvenient to start Airsonic after every restart of your computer/server by yourself, you can setup a Windows Service. This Windows Service will start Airsonic automatically when you boot your computer, you don't even need to logon.
+
+> **NOTE**: To run a Java application as a Windows Service there would be a [Java Service Wrapper](https://wrapper.tanukisoftware.com), but it's not really easy to setup. That's why we go for NSSM.
+
+Create a new directory `C:\Program Files\NSSM` and copy `nssm.exe` to this folder.
+
+Open a command prompt or the PowerShell console and change the directory to the NSSM folder `cd C:\Program Files\NSSM`.
+
+Type `nssm install Airsonic` to open the NSSM service installer and start setting up the Windows Service for Airsonic.
+
+Set the following settings in the `Application` tab.
+
+- Path = `java.exe`
+- Startup directory = `C:\Program Files\Airsonic`
+- Arguments = `-Dairsonic.home="C:\\Program Files\\Airsonic" -Dserver.port=4040`
+
+Open the `Shutdown` tab and remove all the flags in the checkboxes excepted the one for `Generate Contral-C`. Set the timeout to `300000`.
+
+> **NOTE**: This very long timeout (5 minutes) is needed, as Airsonic takes some time to shutdown properly.
+
+That's it, you're ready to start the Windows Service by typing `net start Airsonic` in your command prompt or PowerShell console.
+
+Wait some seconds and browse to `http://localhost:4040`. You should see the Airsonic logon page.
+
+##### Run the Airsonic Windows Service under a Service Account
+
+The Airsonic Windows Service we just setup, is now running under the `SYSTEM` account. This is just fine, if all your media are on a local disk. But if you have your media on a network share, e.g. on a NAS you need to be able to connect to this share. To access this share you need to have the propper permissions. To achieve that, you should run your Airsonic under a service account and not by using the `SYSTEM` account. In this instructions this service account is called `sa-airsonic`.
```

Comment:
:+1: from my understanding `SYSTEM` is very close to being an admin on Windows right? If so, running as another user is a very good thing!

Reply:
Yes that's true! I also added a line about this:
_Running the Airsonic Windows Service using another (non-admin) user also improves the safety of your system, as the `SYSTEM` account has way more rights then really needed to run Airsonic._